compliance/collection/nodeinventorizer/node_analyzer.go

```diff
@@ -63,6 +63,10 @@ func (n *NodeAnalyzer) Scan(nodeName string) (*storage.NodeInventory, error) {
 		Notes:      []storage.NodeInventory_Note{storage.NodeInventory_LANGUAGE_CVES_UNAVAILABLE},
 	}
 
+	if len(protoComponents.GetRhelComponents()) == 0 {
+		m.Notes = append(m.Notes, storage.NodeInventory_CONTENT_SETS_UNAVAILABLE)
+	}
+
```

Comment:
I don't think we need this note here: The list of content sets is empty, isn't that enough to determine that content sets are unavailable?

Reply:
I don't quite understand your comment. Isn't determining that the content sets are unavailable because the list of components is empty exactly what this line is doing? And then writing the corresponding note.

Reply:
The note `NodeInventory_CONTENT_SETS_UNAVAILABLE` seems not to be used in other places in this code. Do we need it?

Reply:
We actually don't, you two were right. It's removed now.